<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataPDF</class>
 <widget class="QMainWindow" name="MetadataPDF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>346</width>
    <height>455</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MetadataPDF</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/resources/icons/logo.png</normaloff>:/resources/icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>346</width>
     <height>394</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="buttonConvertMetadata">
          <property name="text">
           <string>Convert Metadata</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/icons/string.png</normaloff>:/resources/icons/string.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonHelp">
          <property name="text">
           <string>Help</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/icons/questionmark.png</normaloff>:/resources/icons/questionmark.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="buttonExit">
        <property name="text">
         <string>Exit program</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/resources/icons/exit.png</normaloff>:/resources/icons/exit.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QPushButton" name="buttonSelectPDF">
          <property name="text">
           <string>Select PDF</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/resources/icons/load.png</normaloff>:/resources/icons/load.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="linePDFFile">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelPDFOutput">
          <property name="text">
           <string>PDF Output</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="linePDFOutput">
          <property name="text">
           <string>output.pdf</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelTitle">
          <property name="text">
           <string>Title</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineTitle">
          <property name="text">
           <string>Document</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labelAuthor">
          <property name="text">
           <string>Author</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineAuthor">
          <property name="text">
           <string>NameAuthor</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="labelSubject">
          <property name="text">
           <string>Subject</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="lineSubject">
          <property name="text">
           <string>Subject</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="labelKeywords">
          <property name="text">
           <string>Keywords</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLineEdit" name="lineKeywords">
          <property name="text">
           <string>Replace, fix</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="labelModDate">
          <property name="text">
           <string>ModDate</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QDateTimeEdit" name="dateMod">
          <property name="displayFormat">
           <string>dd/MM/yyyy HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="labelCreationDate">
          <property name="text">
           <string>CreationDate</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QDateEdit" name="dateCreation">
          <property name="displayFormat">
           <string>dd/MM/yyyy HH:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="17" column="0">
         <widget class="QLabel" name="labelCreator">
          <property name="text">
           <string>Creator</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLineEdit" name="lineCreator">
          <property name="text">
           <string>MetadataPDF</string>
          </property>
         </widget>
        </item>
        <item row="19" column="0">
         <widget class="QLabel" name="labelProducer">
          <property name="text">
           <string>Producer</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QLineEdit" name="lineProducer">
          <property name="text">
           <string>MetadataPDF</string>
          </property>
         </widget>
        </item>
        <item row="18" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="16" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="14" column="1">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="12" column="1">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="1">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="1">
         <spacer name="verticalSpacer_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_10">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_11">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_12">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/logo.png</normaloff>:/resources/icons/logo.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/questionmark.png</normaloff>:/resources/icons/questionmark.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/resources/icons/exit.png</normaloff>:/resources/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buttonSelectPDF</tabstop>
  <tabstop>linePDFFile</tabstop>
  <tabstop>linePDFOutput</tabstop>
  <tabstop>lineTitle</tabstop>
  <tabstop>lineAuthor</tabstop>
  <tabstop>lineSubject</tabstop>
  <tabstop>lineKeywords</tabstop>
  <tabstop>lineCreator</tabstop>
  <tabstop>lineProducer</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
